pkg/adapter/adapter.go

```diff
@@ -93,13 +94,23 @@ func (s *sinker) createRecorder(ctx context.Context, agentName string) record.Ev
 }
 
 func (s *sinker) Start(ctx context.Context) error {
+
 	// Create EventListener Sink
 	r := sink.Sink{
-		KubeClientSet:          kubeclient.Get(ctx),
-		DiscoveryClient:        s.Clients.DiscoveryClient,
-		DynamicClient:          dynamicclient.Get(ctx),
-		TriggersClient:         s.Clients.TriggersClient,
-		HTTPClient:             http.DefaultClient,
+		KubeClientSet:   kubeclient.Get(ctx),
+		DiscoveryClient: s.Clients.DiscoveryClient,
+		DynamicClient:   dynamicclient.Get(ctx),
+		TriggersClient:  s.Clients.TriggersClient,
+		HTTPClient: &http.Client{
+			Transport: &http.Transport{
```

Comment:
Should we use the values from here? https://github.com/tektoncd/triggers/blob/main/config/controller.yaml#L59-L62

Reply:
Yes, I changed to that.